### PR TITLE
Add missed amazonec2-ssh-port

### PR DESCRIPTION
**Pull request by axw-pivorra**

### CODE REVIEW

**Comment by axw-pivorra** [comment]
Signed-off-by: Philippe IVORRA <pivorra@axway.com>

**Comment by dgageot** [comment]
@axw-pivorra please sign your commit using those instructions:
```
$ git clone -b "master" git@github.com:axw-pivorra/machine.git somewhere
$ cd somewhere
$ git commit --amend -s --no-edit
$ git push -f
```

File: drivers/amazonec2/amazonec2.go [hunk 1]
```diff
@@ -1107,14 +1124,14 @@ func (d *Driver) configureSecurityGroupPermissions(group *ec2.SecurityGroup) ([]
...
-			IpProtocol: aws.String("tcp"),
-			FromPort:   aws.Int64(22),
-			ToPort:     aws.Int64(22),
-			IpRanges:   []*ec2.IpRange{{CidrIp: aws.String(ipRange)}},
-		})
-	}
+        if !hasPorts[fmt.Sprintf("%d/tcp", d.BaseDriver.SSHPort)] {
+                perms = append(perms, &ec2.IpPermission{
+                        IpProtocol: aws.String("tcp"),
+                        FromPort:   aws.Int64(22),
```
Comment (dgageot) [inline drivers/amazonec2/amazonec2.go]: As noted by @axw-pivorra, this should be `aws.Int64(int64(d.BaseDriver.SSHPort)),`

Comment (axw-pivorra) [reply drivers/amazonec2/amazonec2.go]: As it's port range, FromPort and ToPort needs to be set as aws.Int64(int64(d.BaseDriver.SSHPort))
Correction to apply on line 1130

**Comment by axw-pivorra** [comment]
Fixed by #4350

**Pull request closed without merging**